services/subscriber/service.go

```diff
@@ -285,7 +293,8 @@ func (s *Service) updateSubs(wg *sync.WaitGroup) error {
 				}
 				sub, err := s.createSubscription(se, si.Mode, si.Destinations)
 				if err != nil {
-					return err
+					s.Logger.Printf("Subscription creation failed for: %s", si.Name)
+					continue
```

Comment:
I expected the `CreateFailures` stat to be incremented after a call to `createSubscription` so that one call can increment the stat by at most one. Can you change it so that the stat is incremented only after the call to `createSubscriptions`. 

If we also want to track the failures per destination then a new stats should be added to the `writerStats` type.